src/button/Button.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Types from 'prop-types';
+import classNames from 'classnames';
+import './Button.less';
+
+export const BUTTON_VARIANTS = ['primary', 'pay', 'secondary', 'danger'];
+export const BUTTON_SIZES = ['xs', 'sm', 'md', 'lg'];
+export const BUTTON_STATES = ['default', 'loading', 'disabled'];
+
+const Button = ({ label, buttonState, size, variant, block, onClick }) => {
+  const isDisabled = buttonState === 'disabled';
+  const isLoading = buttonState === 'loading';
+
+  const classes = classNames(`btn btn-${size}`, {
+    'btn-loading': isLoading,
+    'btn-primary': variant === 'primary',
+    'btn-success': variant === 'pay',
+    'btn-default': variant === 'secondary',
+    'btn-danger': variant === 'danger',
+    'btn-block': block,
+  });
+
+  return (
+    <button
+      type="button"
+      className={classes}
+      onClick={e => onClick(e)}
+      disabled={isDisabled || buttonState === 'loading'}
+    >
+      {label}
+      {isLoading && <span className={`btn-loader ${block ? '' : 'm-l-2'}`} />}
```

Comment:
What would you think about using `classNames` here?
```suggestion
      {isLoading && <span className={classNames('btn-loader', { 'm-l-2': block })} />}
```

Reply:
fixed